packages/lesswrong/server/manualMigrations/2022-12-13-defaultSessionReplay.ts

```diff
@@ -0,0 +1,15 @@
+import { registerMigration, fillDefaultValues } from './migrationUtils';
+import Users from '../../lib/collections/users/collection';
+
+
+registerMigration({
+  name: "defaultSessionReplay",
+  dateWritten: "2022-12-13",
+  idempotent: true,
+  action: async () => {
+    await fillDefaultValues({
+      collection: Users,
+      fieldName: "allowDatadogSessionReplay",
+    });
+  },
+});
```

Comment:
Do we need this if the field is optional and the default value is `false` anyway?

Reply:
I guess not, I'll remove it